src/Domain/Person.php

```diff
@@ -170,8 +172,15 @@ public function validateCaptchaAndRawPasswordSetIfNew(ExecutionContextInterface
         }
 
         // Entity brand new or somehow otherwise without a password, and none set.
-        if (empty($this->password) && empty($this->raw_password)) {
-            $context->buildViolation('Password is required to create an account')
+        $passwordMissing = (
```

Comment:
@NoelLH I'd probably rename this to something a bit more accurate like `$passwordMissingOrBlank` or `$passwordInvalid` or `$passwordMissingOrDoesNotMeetRequirements`.